packages/notification-center/src/store/novu-theme.context.ts

```diff
@@ -1,10 +1,79 @@
 import React from 'react';
 import { ColorScheme } from '../index';
+import { ICommonTheme } from './novu-theme-provider.context';
 
 export interface IThemeContext {
   colorScheme: ColorScheme;
+  theme: INovuTheme;
+  common: ICommonTheme;
 }
 
 export const ThemeContext = React.createContext<IThemeContext>({
   colorScheme: 'light',
+  theme: null,
+  common: null,
 });
+
+export interface INovuTheme {
+  layout?: IThemeLayout;
+  header?: IThemeHeader;
+  popover?: IThemePopover;
+  notificationListItem?: IThemeNotificationListItem;
+  footer?: IThemeFooter;
+  mainColor?: string;
```

Comment:
Can we remove this one? Looks like it's only used for the loader animation as a fallback color, we can maybe have a specific category for the loader.